lib/containers/runtime.pm

```diff
@@ -0,0 +1,96 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Test Docker’s btrfs storage driver features for image and container management
+# Maintainer: qac team <qa-c@suse.de>
+
+package containers::runtime;
+use Mojo::Base -base;
+use testapi;
+use Test::Assert 'assert_equals';
+
+has runtime => undef;
+
+sub _rt_assert_script {
```

Comment:
Don't you want to remove the trailing underscore from the begining of those functions? We don't have it on public cloud either.

Reply:
cc: @asmorodskyi @grisu48

Reply:
This functions are private to the class. they should never called directly. see also https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/11307#discussion_r513308703 what is their purpose.

Reply:
We have this approach in public cloud too and we don't have the trailing underscore.

Reply:
i do not know about the other approaches but `_` prefix is valid perl convention to define a method private. So i do not think it needs to change.

Reply:
Personally, I am not keen on `rt` as it can ambiguous in our world.  AFAICT  `_` is not a problem as it is just a label and nothing more.

Reply:
i dont know what to say. Can i say that this is too picky?

Reply:
OK, if we don't plan to call those methods outside of the file I'm ok with `_` at the beginning and I'm sorry that I insisted on removing it. My impression was that you plan to call those methods outside of the package as well.

Reply:
I agree with @mloviska. I don't even know what `rt_` means. Is it randomly generated prefix?

Also @asmorodskyi suggested to remove the `rt_` and his suggestion was marked as 'resolved'.

Reply:
> i dont know what to say. Can i say that this is too picky?

Ofc, no one is going to force you to remove `rt` prefix, if this is your concern. :) It is just ambiguous IT abbreviation in general, on the other hand the methods are already packed in `runtime.pm` making the prefix a bit superfluous to me.
Sometimes less is more :)

Reply:
i think @mloviska put it nicely. we are arguing about naming and microstuff. personally i make totally sense of it. i would like to resolve this topic and personally if someone goes ahead and change the names i will approve it.

Reply:
> i think @mloviska put it nicely. we are arguing about naming and microstuff. personally i make totally sense of it. i would like to resolve this topic and personally if someone goes ahead and change the names i will approve it.

Yup, not really so serious :)

Reply:
OK, can we at least know what does `rt_` mean?

Reply:
> i dont know what to say. Can i say that this is too picky?

good enough for me :) feel free to ignore it in this case . No hard feeling from my side :) I am 100% support your approach that if I really dislike something I can come up with my own PR to "fix" it. I was just trying to convince you in my POV sorry if I did this too intensive. 
Please apply this comment to **all** my suggestions in this and other PRs